onelogin/get.go

```diff
@@ -216,3 +198,45 @@ func Get(app, provider string) (*awsprovider.Credentials, error) {
 
 	return &creds, nil
 }
+
+func getDevice(devices GenerateSamlAssertionResponseDevices) (deviceID, deviceType string, err error) {
+	// we might have more than one device
```

Comment:
I would remove this comment and instead add proper documentation for the function (`// getDevice receives a slice of Devices, prompts the user to select one and returns the selected device or an error`).